src/Mod/Fem/femtaskpanels/task_material_common.py

```diff
@@ -567,20 +567,18 @@ def update_material_property(self, inputfield_text, matProperty, qUnit, variatio
             old_value = Units.Quantity(self.material[matProperty]).Value
             # value = float(inputfield_text)  # this fails on locale with comma
             # https://forum.freecad.org/viewtopic.php?f=18&t=56912&p=523313#p523313
-        if value:
```

Comment:
Why do we even check if `Base::Quantity`? It's not a pointer, so it can't be a nullptr? Is there even a reason with this check? Can we just remove it completely?